.setup/bin/populate_office_hours_queue.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+
+import argparse
+from sqlalchemy import create_engine, Table, MetaData, select, update, insert
+import random
+import os
+from datetime import datetime, timedelta
+from submitty_utils import dateutils
+
+DB_HOST = "localhost"
+DB_PORT = 5432
+DB_USER = "submitty_dbuser"
+DB_PASS = "submitty_dbuser"
+
+def main():
+    # get current semester
+    term_id = dateutils.get_current_semester()
+
+    # set up the database connection
+    database = "submitty_" + term_id + "_sample"
+    engine = create_engine("postgresql:///{}?host={}&port={}&user={}&password={}"
+                           .format(database, DB_HOST, DB_PORT, DB_USER, DB_PASS))
+    conn = engine.connect()
+    metadata = MetaData(bind=engine)
+    queues_table = Table("queue_settings", metadata, autoload=True)
+    queue_entries_table = Table("queue", metadata, autoload=True)
+    users_table = Table("users", metadata, autoload=True)
+
+    # Find all queues in course
+    res = conn.execute(select(queues_table))
+    queues_lookup = {x["code"]: x for x in res}
+    res.close()
+
+    # finds all open queues
+    all_queues = [x for x in queues_lookup if queues_lookup[x]["open"]]
+
+    # if there are no open queues, don't populate anything
+    if len(all_queues) == 0:
+        return
+
+    # Find all students in class
+    res = conn.execute(select(users_table))
+    all_users = res.fetchall()
+    res.close()
+
+    # Find all ids of students and graders
+    all_student_ids = [x['user_id'] for x in all_users if x['user_group'] == 4 and x['registration_section'] != None]
```

Comment:
It's generally preferable to use `is`/`is not` when comparing something to `None` in Python.

```suggestion
    all_student_ids = [x['user_id'] for x in all_users if x['user_group'] == 4 and x['registration_section'] is not None]
```